ui/packages/app/grafana-datasource-plugin/src/plugin.json

```
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
+  "type": "datasource",
+  "name": "Parca Datasource",
```

```suggestion
  "name": "Parca",
```
All the other data sources are simply called by their name only.